tap_kafka/__init__.py

```diff
@@ -57,11 +63,14 @@ def get_args():
 def main_impl():
     """Main tap-kafka implementation"""
     args = get_args()
-
     kafka_config = {'topic': args.config['topic'],
                     'group_id': args.config['group_id'],
                     'bootstrap_servers': args.config['bootstrap_servers'].split(','),
-                    'encoding': args.config.get('encoding', 'utf-8'),
+                    'consumer_timeout_ms': args.config.get('consumer_timeout_ms', DEFAULT_CONSUMER_TIMEOUT_MS),
+                    'session_timeout_ms': args.config.get('session_timeout_ms', DEFAULT_SESSION_TIMEOUT_MS),
+                    'heartbeat_interval_ms': args.config.get('heartbeat_interval_m', DEFAULT_HEARTBEAT_INTERVAL_MS),
```

Comment:
you have a missing `s` in the `args.config.get('heartbeat_interval_m'`

Reply:
thank you. Fixed by 9be8842a7316167c674cad060c04060914da6d4f and added tests to avoid same problem in the future